### ISSUE TITLE
rule one-var fix to incorrect code when using  semicolon-less style

### ISSUE THREAD

**Issue by Aladdin-ADD** [eslint/eslint#10330]
**Tell us about your environment**

* **ESLint Version:** master
* **Node Version:** n
* **npm Version:** n

**What parser (default, Babel-ESLint, etc.) are you using?**
default
**Please show your full configuration:**
```js
{
  "one-var": 2
}
```

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
var i
var j
```

**What did you expect to happen?**
it should be fixed to
```js
var i, j
```
**What actually happened? Please include the actual, raw output from ESLint.**
```js
var ,
 j
```

**Comment by malcolmsgroves** [comment]
I'll work up a PR for this
